src/successModal/SuccessModal.style.tsx

```diff
@@ -34,7 +34,7 @@ const StyledSuccessModal = styled(Modal)`
 `
 
 /* LAYOUT: Media */
-const Media = styled.div`
+export const Media = styled.div`
```

Comment:
IMHO we should export an **StyledSuccessModal** object to avoid having all components as pure components since they are part of **SuccessModal**

Reply:
How can I do that?

Reply:
Ah! or do you mean to keep it as before with the 
```
export const SuccessModalElements = {
  StyledSuccessModal,
  Media,
  Figure,
  Content,
  SuccessTitle,
  SuccessAction,
  SuccessButton,
}
```